Doc/library/sys.rst

```diff
@@ -1261,6 +1261,11 @@ always available.
         This is still called as a fallback if a :data:`meta_path` entry doesn't
         have a :meth:`~importlib.abc.MetaPathFinder.find_spec` method.
 
+    .. versionchanged:: 3.12
+
+        :meth:`!find_module` has been removed and wouldn't be called as a
```

Comment:
I think we should do the reverse thing: explain what happens now and in `.. versionchanged:: 3.12` we can add a note about how it is used to be.

Reply:
The main part of `sys.meta_path` describes what the current behavior is. In my understanding, you mean to remove the `find_path` in `.. versionchanged:: 3.4` and move it to `.. versionchanged:: 3.12` to show how it used to be?